### PR TITLE
P1/feat/103 hover on card to show category banner

### PR DESCRIPTION
**Pull request by netlight-saraandersson**
This is not the best solution, so if anyone can figure out a better way to handle this it would be great :D

### CODE REVIEW

**Comment by elinaolsson** [comment]
I think it looks good! At least I can't come up with a better solution on top of mind. However I think we also should implement this on the Kanban cards to be consistent with the design ☘️

File: app/assets/main/react_components/routes/dashboard/climate_actions/carousel/CarouselCard.jsx [hunk 1]
```diff
@@ -55,9 +55,13 @@ const CarouselCard = ({
...
-    <div className="flex flex-1 min-h-full transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-102">
-      <div className="min-h-full pt-20 flex m-lg:pt-24 justify-evenly">
-        <div className="border-gray-tint-2 rounded-lg shadow-lg pb-2 mx-2 flex flex-col bg-white">
+    <div
+      className={`flex flex-1 min-h-full transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-102`}
+    >
+      <div className={`min-h-full pt-20 flex m-lg:pt-24 justify-evenly`}>
+        <div
+          className={`group border-gray-tint-2 rounded-lg shadow-lg pb-2 mx-2 flex flex-col bg-white`}
+        >
```
Comment (netlight-johannadagfalk) [inline app/assets/main/react_components/routes/dashboard/climate_actions/carousel/CarouselCard.jsx]: Is this special syntax with {` `} needed?

File: app/assets/main/react_components/routes/dashboard/climate_actions/manage_actions/kanban/KanbanCard.jsx [hunk 2]
```diff
@@ -89,6 +92,16 @@ const KanbanCard = ({
...
+                    categoryColorHover={getCategoryColor(
+                      'category_' +
+                        categoryName(
+                          categories,
+                          item.climate_action_category_id
+                        )
+                          .toLowerCase()
+                          .replace(/ /g, '_') +
+                        '_active'
+                    )}
```
Comment (netlight-johannadagfalk) [inline app/assets/main/react_components/routes/dashboard/climate_actions/manage_actions/kanban/KanbanCard.jsx]: For consistency - should we move the calculation outside of the div?

File: app/assets/main/react_components/routes/dashboard/helpers/CategoryColorHelper.js [hunk 3]
```diff
@@ -7,3 +7,23 @@ export const categoryName = (categories, itemID, isAchievement) => {
...
+  } else if (category_tag == 'category_public_emissions_active') {
+    return 'bg-gray-accent';
+  } else if (category_tag == 'category_car_active') {
+    return 'bg-orange-accent';
+  } else if (category_tag == 'category_shopping_active') {
+    return 'bg-pink-accent';
+  } else if (category_tag == 'category_housing_active') {
+    return 'bg-green-accent';
+  }
+};
```
Comment (netlight-johannadagfalk) [inline app/assets/main/react_components/routes/dashboard/helpers/CategoryColorHelper.js:29]: Dont we already have this in the css file?

Comment (netlight-saraandersson) [reply app/assets/main/react_components/routes/dashboard/helpers/CategoryColorHelper.js:29]: The group-hover attribute does not work with @apply, therefore I created a new file where only the color is returned and not the class.

**Review by netlight-johannadagfalk — APPROVED** [review]

**Review by elinaolsson — APPROVED** [review]
Great job! 🎖️

**Pull request merged**